pkgs/development/python-modules/pytest-subtests/default.nix

```diff
@@ -18,6 +19,8 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ setuptools-scm ];
 
+  propagatedBuildInputs = [ pytest ];
```

Comment:
pytest extensions should not propagate pytest so it is possible to use them with different versions of pytest